asciidoc/cypher-workflow.adoc

```diff
@@ -245,28 +259,31 @@ include::{javascript-examples}/examples.test.js[tags=read-write-transaction]
 
 ======
 
-// [.include-with-python]
-// ======
-// [source, python]
-// ----
-// include::{python-examples}/read_write_transaction_example.py[tags=read-write-transaction]
-// ----
-// ======
+[.include-with-python]
+======
+[source, python]
+----
+include::{python-examples}/read_write_transaction_example.py[tags=read-write-transaction]
+----
+======
```

Comment:
This example is unresolved, and is also missing a title in the same fashion as the other languages?

Reply:
fixed